docs/getting-started/first-project.md

```diff
@@ -0,0 +1,327 @@
+1. Set your font family variable by equaling your font name to `$g-font-family-primary` variable. Then add `$g-font-family-primary` variable to `$g-font-family-sans-serif` as the first value.
+
+```scss
+$g-font-family-primary: 'Roboto';
+$g-font-family-sans-serif: $g-font-family-primary, 'Helvetica Neue', Helvetica, Arial, sans-serif !default;
+```
+
+> The `$g-font-family-sans-serif` variable sets body's font family.
+
+2. Font sizes are customizable but for the sake of example we are choosing not to.
+
+```scss
+$g-font-size-h1: px-to-rem(36px) !default;
+$g-font-size-h2: px-to-rem(30px) !default;
+$g-font-size-h3: px-to-rem(24px) !default;
+$g-font-size-h4: px-to-rem(18px) !default;
+$g-font-size-h5: 1rem !default;
+$g-font-size-h6: px-to-rem(14px) !default;
+```
+
+3. Other heading variables are also customizable.
+
+```scss
+$g-headings-font-family: inherit !default;
+$g-headings-font-weight: 500 !default;
+$g-headings-line-height: 1.1 !default;
+$g-headings-margin-bottom: 1rem !default;
+```
+
+> Headings and other html elements can be customized on `base/_base.scss` file. (For example: You can use a customization as shown here to set different line-height values for each heading tags. )
+
+4. Global margin(`$g-gaps`) and padding(`$g-pads`) variables can be customized or increased based on the design's needs. (2xlarge value is added to the map.)
+
+```scss
+$g-gaps: (
+  normal: $g-base-gap,
+  small: (
+    $g-base-gap / 1.5,
+  ),
+  xsmall: (
+    $g-base-gap / 3,
+  ),
+  2xsmall: (
+    $g-base-gap / 6,
+  ),
+  medium: (
+    $g-base-gap * (4 / 3),
+  ),
+  large: (
+    $g-base-gap * (5 / 3),
+  ),
+  xlarge: (
+    $g-base-gap * 2,
+  ),
+  2xlarge: (
+    $g-base-gap * 4,
+  ),
+);
+```
+
+> When adding a new margin or padding value, please ensure that it's a children of `$g-base-gap` variable.
+
+## Creating A Component
+
+Start developing components after setting the global variables.
+
+> The following steps use predefined card component from the `c-card.scss` file.
+
+1. Create a folder `c-card` in the components folder and open the scss file `c-card.scss`.
+2. Develop the card component as shown below.
+
+```html
+<div class="c-card">
+  <div class="c-card__header">
+    This is a header
+  </div>
+  <img src="https://via.placeholder.com/150" alt="Placeholder" class="c-card__image" />
+  <div class="c-card__content">
+    <h4 class="u-margin-top-0 u-margin-bottom-xsmall">This is a card.</h4>
+    <p class="u-margin-bottom-0">
+      Lorem ipsum dolor sit amet consectetur, adipisicing elit. Earum eveniet illum aut harum ullam
+      dolore repudiandae pariatur optio exercitationem.
+    </p>
+  </div>
+</div>
+```
+
+> Since we don't want the tags on **card content** element to be dependent to **the card component**, we have zeroed default margins of h4 and p tags by using utility classes. On the case of not using utility classes; we would need to add `c-card__title` class to h4 element and `c-card__text` class to p element.
```

Comment:
```suggestion
> Since we don't want the tags on **card content** element to be dependent to **the card component**, we have cleared default margins of `h4` and `p` tags by using utility classes. On the case of not using utility classes; we would need to add `c-card__title` class to `h4` element and `c-card__text` class to `p` element.
```